modules/core/src/v2/coins/btc.ts

```diff
@@ -85,17 +85,23 @@ export class Btc extends AbstractUtxoCoin {
     }).call(this);
   }
 
+  // using blockchair api: https://blockchair.com/api/docs#link_300
+  // https://api.blockchair.com/{:btc_chain}/dashboards/address/{:address}₀
+  // example utxo from response:
+  // {"block_id":-1,"transaction_hash":"cf5bcd42c688cb7c55b5811645e7f0d2a000a85564ca3d6b9fc20f57e14b30bb","index":1,"value":558},
```

Comment:
implementation comment should go inside func

Reply:
thanks, done!